src/store/reducers.js

```diff
@@ -32,6 +33,10 @@ function entitiesLoaded(state, { payload }) {
 }
 
 function entityLoaded(state, { payload: { entitlements } } = { payload: {} }) {
+
+    const hasRosCookie = insights.chrome.visibilityFunctions.isProd() ?
+        insights.chrome.visibilityFunctions.hasCookie('cs_ros_beta_enable') : true;
```

Comment:
This one should also pass in the value
```suggestion
        insights.chrome.visibilityFunctions.hasCookie('cs_ros_beta_enable', '1') : true;
```

Reply:
Thank you! Good catch :+1: